File: app/src/main/java/com/test/demo/MainActivity.java
package com.test.demo;

import android.os.Bundle;
import android.util.Log;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import com.test.demo.databinding.ActivityMainBinding;
import com.vpn.lib.VPNInit;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
        VPNInit.INSTANCE.setVip(true); // Set VIP mode after purchasing
        binding.start.setOnClickListener(v -> VPNInit.INSTANCE.startVpn(MainActivity.this));
    }

    @Override
    protected void onResume() {
        super.onResume();
    }
}

File: app/src/main/java/com/test/demo/MainActivity.kt
package com.test.demo
import android.os.Bundle
import android.util.Log
import androidx.activity.ComponentActivity
import com.test.demo.databinding.ActivitiyMainBinding
import com.vpn.lib.VPNInit

class MainActivity : ComponentActivity() {
    private lateinit var binding: ActivitiyMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitiyMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
        VPNInit.setVip(true)//购买VIP后设置true
        binding.start.setOnClickListener {
            VPNInit.startVpn(this)
        }
    }

    override fun onResume() {
        super.onResume()

    }


}